packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -76,32 +79,10 @@ class NetworkController extends DisposableController
   void _processHttpProfileRequests({
     required int timelineMicrosOffset,
     required List<HttpProfileRequest> httpRequests,
```

Comment:
are the httpRequests the new requests here? if so maybe we can name newHttpRequests to make that clear and to contrast with currentRequests

Reply:
they are new or updated requests so I will go for `newOrUpdatedHttpRequests`